src/backend/distributed/sql/udfs/citus_stats_tenants/11.3-1.sql

```diff
@@ -1,27 +1,66 @@
-CREATE OR REPLACE FUNCTION pg_catalog.citus_stats_tenants(
+-- cts in the query is an abbreviation for citus_stats_tenants
+CREATE OR REPLACE FUNCTION pg_catalog.citus_stats_tenants (
     return_all_tenants BOOLEAN DEFAULT FALSE,
+    OUT nodeid INT,
     OUT colocation_id INT,
     OUT tenant_attribute TEXT,
     OUT read_count_in_this_period INT,
     OUT read_count_in_last_period INT,
     OUT query_count_in_this_period INT,
     OUT query_count_in_last_period INT,
-    OUT score BIGINT)
-RETURNS SETOF RECORD
-LANGUAGE C
-AS 'citus', $$citus_stats_tenants$$;
-
+    OUT score BIGINT
+)
+    RETURNS SETOF record
+    LANGUAGE plpgsql
+    AS $function$
+BEGIN
+    RETURN QUERY
+    SELECT *
```

Comment:
cool, similar trick with citus_stat_activity  :) I wish we had a single function to pull from different tables, but this is also good